src/main/java/hanieum/conik/domain/user/dto/UserSignUpRequest.java

```diff
@@ -0,0 +1,10 @@
+package hanieum.conik.domain.user.dto;
+
+import hanieum.conik.domain.user.UserRole;
+import jakarta.validation.constraints.Email;
+
+public record UserSignUpRequest(@Email String email,
+                                String password,
+                                String phoneNumber,
+                                Boolean termsOfServiceAgreed,
+                                UserRole role) {}
```

Comment:
```suggestion
public record UserSignUpRequest(
    @Email 
    String email,
                               
    String password,

    String phoneNumber,

    Boolean termsOfServiceAgreed,

    UserRole role
) {}
```
이런식으로 작성하면 가독성이 좀 더 좋을 것 같아요